Assets/Colyseus/Runtime/Scripts/Managers/ColyseusManager.cs

```diff
@@ -84,6 +83,7 @@ private void InitializeInstance()
             }
 
             Instance = GetComponent<T>();
+            _colyseusSettings = ScriptableObject.CreateInstance<ColyseusSettings>();
```

Comment:
Originally the settings object was designed to be able to be assigned in editor so devs could have multiple settings objects (local, dev, staging, prod, etc). @lpsandaruwan can you confirm that this doesn't break that functionality? Or if it does/needs to thats ok too, we just need to be able to provide an explanation/workaround for people who use that feature.